Cannot run tests on macOS

Running the following on macOS causes a hang

```
python -Wd  -m pytest tests --doctest-modules wikidict
```

I tried to run 

```
python -Wd  -m pytest tests -s --doctest-modules wikidict
```

and I got more info : 

```
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/Cellsar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/pool.py", line 114, in worker
    task = get()
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/queues.py", line 368, in get
    return _ForkingPickler.loads(res)
  File "/Users/ebook-reader-dict/wikidict/render.py", line 44, in <module>
    MANAGER = multiprocessing.Manager()
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/context.py", line 57, in Manager
    m.start()
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/managers.py", line 553, in start
    self._process.start()
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 118, in start
    assert not _current_process._config.get('daemon'), \
AssertionError: daemonic processes are not allowed to have children
```


I could trace it back to this commit :
https://github.com/BoboTiG/ebook-reader-dict/commit/9c714631269203d42f77759d6dcd1f4be3869360

@BoboTiG is MISSING_TPL_SEEN useful ? did you use it since you implemented it ?

--- a/wikidict/render.py
+++ b/wikidict/render.py
@@ -40,10 +40,10 @@ wikitextparser._spans.WIKILINK_PARAM_FINDITER = lambda *_: ()
 
 Sections = Dict[str, wtp.Section]
 
-# Multiprocessing shared globals
-MANAGER = multiprocessing.Manager()
+# Multiprocessing shared globals, init in render() see #1054
+MANAGER = ""
 LOCK = multiprocessing.Lock()
-MISSING_TPL_SEEN: List[str] = MANAGER.list()
+MISSING_TPL_SEEN: List[str] = []
 
 
 def find_definitions(
@@ -362,6 +362,8 @@ def render(in_words: Dict[str, str], locale: str) -> Words:
         if any(head_section in code for head_section in sections)
     }
 
+    MANAGER = multiprocessing.Manager()
+    MISSING_TPL_SEEN: List[str] = MANAGER.list()  # noqa
     results: Words = MANAGER.dict()
 
     with multiprocessing.Pool(processes=multiprocessing.cpu_count()) as pool:

